src/PackageResolver.php

```diff
@@ -98,7 +98,7 @@ private function parseVersion(string $package, string $version, bool $isRequire)
             try {
                 $config = @json_decode(file_get_contents(Factory::getComposerFile()), true);
             } finally {
-                if (!$isRequire || !($config['extra']['symfony']['require'] || isset($config['require']['symfony/framework-bundle']))) {
+                if (!$isRequire || !(isset($config['extra']['symfony']['require']) || isset($config['require']['symfony/framework-bundle']))) {
```

Comment:
That does not mean the same thing. Here, is it is defined, we still need to check if it's true or false. Can you update the patch?